config.yaml

```diff
@@ -174,6 +178,19 @@ res_target_2030:
   PT: 0.8  # p.41 Table 9 https://energy.ec.europa.eu/system/files/2020-06/pt_final_necp_main_en_0.pdf
   # SE:
 
+# Electrolyzer targets in GW_el
+h2_target_2030:
+  DE: 10
```

Comment:
I also tried running it with country subsets. Running with just DE or just AT did work. Running with both did not...